x-pack/plugins/apm/server/routes/alerts/rule_types/transaction_duration/register_transaction_duration_rule_type.test.ts

```diff
@@ -65,7 +65,7 @@ describe('registerTransactionDurationRuleType', () => {
         'Avg. latency is 5,500 ms in the last 5 mins for opbeans-java. Alert when > 3,000 ms.',
       transactionType: 'request',
       serviceName: 'opbeans-java',
-      threshold: 3000000,
+      threshold: 3000,
```

Comment:
We did the reverse change in https://github.com/elastic/kibana/pull/112366. Please double check why we did so and if this change breaks something.

As discussed: I'd love to have  an api test (or e2e test) that reproduces this problem.

Reply:
>We did the reverse change in #112366

I don't think we should show the alerts in microseconds, since the only option we offer when creating an alert is a threshold in milliseconds.

Reply:
What was the reason for the change in https://github.com/elastic/kibana/pull/112366?

Reply:
@sqren looks like what was done in this PR (https://github.com/elastic/kibana/pull/112366) was the opposite of what Cyrille was asking on this [issue](https://github.com/elastic/kibana/issues/112245). He clearly says that the alert should **NOT** be shown with microseconds, but instead in milliseconds, which is what I'm changing in this PR.

Reply:
All right. Change looks fine. Did you find a way to test this?

Reply:
I'll look into it during test plan.